.travis.yml

```diff
@@ -4,34 +4,20 @@ cache:
   directories:
   - $HOME/.cache/pip
   - $HOME/build/RasaHQ/rasa_nlu/data
-  - $HOME/google-cloud-sdk/
 python:
 - '2.7'
 - '3.5'
 - '3.6'
 install:
-- sudo apt-get update
-- if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
-  -O miniconda.sh; else wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-  -O miniconda.sh; fi
-- bash miniconda.sh -b -f -p $HOME/miniconda
-- export PATH="$HOME/miniconda/bin:$PATH"
-- hash -r
-- conda config --set always_yes yes --set changeps1 no
-- conda update -q conda
-- conda info -a
-- conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION scipy=0.19.0 numpy=1.13.1
-  scikit-learn=0.18.1
-- source activate test-environment
 - pip install git+https://github.com/tmbo/MITIE.git
 - pip install -r alt_requirements/requirements_dev.txt
 - pip install -e .
-- pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-1.2.0/en_core_web_sm-1.2.0.tar.gz
+- pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.0.0/en_core_web_md-2.0.0.tar.gz
   --no-cache-dir > jnk
-- python -m spacy link en_core_web_sm en
-- pip install https://github.com/explosion/spacy-models/releases/download/de_core_news_md-1.0.0/de_core_news_md-1.0.0.tar.gz
+- python -m spacy link en_core_web_md en
+- pip install https://github.com/explosion/spacy-models/releases/download/de_core_news_sm-2.0.0/de_core_news_sm-2.0.0.tar.gz
```

Comment:
so have we checked the German model works well? given there's only a small one for that for some reason...

Reply:
I didn't check the models intent classification performance for german / french.

Reply:
well idk - do we want to mention that in the tutorial somewhere?

Reply:
Just checked it on two datasets and there wasn't any difference - which doesn't say there is none but I think we are good to go for now :+1: